app/controllers/users_controller.rb

```diff
@@ -19,14 +20,21 @@ def create
     end
   end
 
-  def show
-    @user = params[:id] ? User.find(params[:id]) : current_user
-    if !current_user
+  def profile
+    if @user
+      @authorized_user = true
```

Comment:
noting to create an issue for determining if we want any user to be able to see any user's information